### PR TITLE
chore: move gnoboard in example/js/expo and simplify the root Makefile

### PR DESCRIPTION
**Pull request by D4ryl00**
Gnoboard uses Expo in its dependencies, so we decided to move it in the `example/js/expo` folder.

The Makefile in the root directory had some rules that compiled the `node_modules` and the `xcworkspace` for Gnoboard, so I removed them to be consistent (that was the only example app that had rules in the root Makefile).

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -44,20 +42,8 @@ PATH := $(GO_BIND_BIN_DIR):$(PATH)
 
 # * Main commands
 
-# `all` and `build` command builds everything (generate, build.ios, build.android)
-all build: generate build.ios build.android
-
-# Build iOS framework
-build.ios: generate framework.ios
```
Comment (jefft0) [inline Makefile:51]: The README still says to use `make build.ios` and `make build.andriod`. Need to update the README?
https://github.com/D4ryl00/gnonative/blob/chore/move-gnoboard/README.md

Comment (D4ryl00) [reply Makefile:51]: fixed, thanks.

File: README.md [hunk 2]
```diff
@@ -9,15 +9,29 @@ These API calls are a programming language-independent wrapper on top of the cor
 
 Watch [this Gno Native Kit tutorial](https://www.youtube.com/watch?v=N1HLyQDHGQ0) to easily get started on building and bringing your (d)apps to mobile and desktop.
 
-## Build instructions
+## Expo module
 
-### Install prerequisites for macOS 13
+To use Gno Native Kit, we advice you to use the Expo module in your Expo React-Native project. Please read the [README](expo/README.md) in the `expo` folder.
```
Comment (jefft0) [inline README.md:14]: "advice" should be "advise"

File: README.md [hunk 3]
```diff
@@ -26,158 +40,78 @@ To install asdf using brew, follow instructions at https://asdf-vm.com . In shor
...
-
-#### Install the tools with asdf (only need to do once)
+### Install the tools with asdf (only need to do once)
 
-```console
+```sh
 make asdf.install_tools
 ```
 
 If you get an error like "https://github.com/CocoaPods/CLAide.git (at master@97b765e) is not yet checked out" then reinstall cocoapods like this:
```
Comment (jefft0) [inline README.md:101]: This error can only happen when building iOS. Previously, it was under the header "Build fo iOS". It's OK if you want to remove the header. Maybe change "If you get an error like" to "On macOS, if you get an error like".

**Review by jefft0 — APPROVED** [review]
In examples/js/expo/gnoboard, tested `make ios` and `make android`. (Nice and simple. Thanks!) Also see my comments on the README.

**Pull request merged**